test/platform/test-platform-identify.ts

```diff
@@ -0,0 +1,124 @@
+/// <reference types="@google/local-home-sdk" />
+import test from 'ava';
+import {
+  injectSmarthomeStubs,
+  MockLocalHomePlatform,
+  ERROR_LISTEN_NOT_CALLED,
+  ERROR_UNDEFINED_VERIFICATIONID,
+  getActiveAppStub,
+  ERROR_UNDEFINED_APP_STUB,
+} from '../../src';
+
+const DISCOVERY_BUFFER: Buffer = Buffer.from('discovery buffer 123');
+const APP_VERSION: string = '0.0.1';
+const DEVICE_ID: string = 'device-id-123';
+const EXECUTE_HANDLER: smarthome.IntentFlow.ExecuteHandler = () => {
+  return {
+    requestId: 'request-id',
+    intent: smarthome.Intents.IDENTIFY,
+    payload: {
+      commands: [
+        {
+          ids: ['123'],
+          status: 'SUCCESS',
+          states: {
+            on: true,
+            online: true,
+          },
+        },
+        {
+          ids: ['456'],
+          status: 'ERROR',
+          errorCode: 'deviceTurnedOff',
+        },
+      ],
+    },
+  };
+};
+
+/**
+ * Inject stubs into smarthome namespace
+ * Assert no active AppStub
+ */
+test.before(t => {
+  injectSmarthomeStubs();
+  t.throws(() => getActiveAppStub(), {
+    instanceOf: Error,
+    message: ERROR_UNDEFINED_APP_STUB,
+  });
+});
+
+/**
+ * Tests that `listen()` was called on the created App.
+ * This is a required flag that indicates handlers have been set.
+ */
+test('trigger-identify-without-listen-throws', async t => {
+  const app: smarthome.App = new smarthome.App(APP_VERSION);
+  const mockLocalHomePlatform = getActiveAppStub().getLocalHomePlatform();
```

Comment:
you don't really need `getActiveAppStub` since you already have the `app` in this context.

Reply:
Done